spec/mailers/supervisor_mailer_spec.rb

```diff
@@ -61,6 +61,26 @@
       create(:case_assignment, casa_case: casa_case, volunteer: volunteer, active: false, updated_at: Date.today - 8.days)
       expect(mail.body.encoded).to_not match("Summary for #{volunteer.display_name}")
     end
+
+    context "when a supervisor has pending volunteer to accepts invitation" do
+      let!(:volunteer2) { create(:volunteer) }
+      let!(:invite) { volunteer2.invite!(supervisor) }
```

Comment:
It doesn't seem you're using `invite`, I mean, it'd probably fit better into a `before` block.